webpack/assets/javascripts/react_app/components/common/ActionButtons/ActionButtons.fixtures.js

```diff
@@ -1,7 +1,7 @@
 import { noop } from '../../../common/helpers';
 
 export const buttons = [
-  { title: 'first', action: { onClick: noop } },
-  { title: 'second', action: { href: 'some-url2', 'data-method': 'put' } },
-  { title: 'third', action: { onClick: noop } },
+  { title: 'first', action: { id: 1, onClick: noop } },
+  { title: 'second', action: { id: 2, href: 'some-url2', 'data-method': 'put' } },
+  { title: 'third', action: { id: 3, onClick: noop } },
```

Comment:
I would add one more button with disabled prop test. 

```suggestion
  { title: 'third', action: { id: 3, onClick: noop } },
  { title: 'fourth', action: { id: 3, onClick: noop, disabled: true } },
```

Reply:
Thanks!